.github/workflows/vm-nightly-test.yml

```diff
@@ -27,37 +23,39 @@ jobs:
     strategy:
       matrix:
         hardfork: ['Byzantium', 'Constantinople', 'Petersburg', 'Istanbul']
-
     steps:
       - uses: actions/setup-node@v1
         with:
           node-version: 8.x
-
       - uses: actions/checkout@v1
-
       - run: npm install
-
-      - name: Run ${{ matrix.hardfork }} state tests
-        run: npm run ${{ format('testState{0}', matrix.hardfork) }}
+      - run: npm run test:state -- --fork=${{matrix.hardfork}}
         env:
           CI: true
 
   test-blockchain:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        hardfork: ['', 'Petersburg']
-
+        hardfork: ['Byzantium', 'Constantinople', 'Petersburg', 'Istanbul']
```

Comment:
I think this overstresses the hardware + run times since blockchain tests are the really heavy ones, let's keep this (respectively change this) to either only `Istanbul` or eventually also `Petersburg`. Blockchain tests are also not getting through, not sure if this is the reason as well.

Reply:
Sounds good, I agree about optimizing our usage.

See edit in opening post, I've simplified the actions to a single run command. I do think the CI test suite should be comprehensive and give us confidence that everything is covered. I'm watching the allForks run now to see its duration, and will report back with its results.